test/cql-pytest/test_counter.py

```diff
@@ -24,6 +24,9 @@ def table2(cql, test_keyspace):
 # Test that the function counterasblob() exists and works as expected -
 # same as bigintasblob on the same number (a counter is a 64-bit number).
 # Reproduces #14742
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #19449")]), "vnodes"],
```

Comment:
Ditto, I think this is the wrong issue.
And same in all other places in this patch.

Reply:
Fixed.